x-pack/test/saved_object_api_integration/common/suites/import.ts

```diff
@@ -163,16 +163,11 @@ export function importTestSuiteFactory(
             type: 'conflict',
             ...(expectedNewId && { destinationId: expectedNewId }),
           };
-          if (fail409Param === 'ambiguous_conflict_1a1b') {
-            // "ambiguous source" conflict
-            error = {
-              type: 'ambiguous_conflict',
-              destinations: [getConflictDest(`${CID}1`)],
-            };
```

Comment:
I forgot to mention:

This was dead code from the draft implementation of #75444; I originally had an "ambiguous source conflict" test case, but we decided to randomize object IDs in that scenario instead of throwing an error. At the time, I changed the individual test cases accordingly, but I forgot to rip out this now-unused portion of code from the common suite.